internal/pkg/mqtt/client_test.go

```diff
@@ -790,6 +791,7 @@ func TestClient_Disconnect(t *testing.T) {
 	if ok {
 		err = nil
 	}
+	assert.Error(t, err)
```

Comment:
```suggestion
	assert.NoError(t, err)
```

Reply:
@lenny-intel 

Actually, if you look at the comment above this, it reads
```
// Expecting a timeout error since creating mqtt client now at the beginning of the Connect() function
```

In other words, the code expects an error to be thrown here.

Reply:
OK, wasn't sure as the comment was clear to me. THX!